pretest/edc-openapi.js

```diff
@@ -1,21 +1,30 @@
-const fs = require('fs/promises');
+const fs = require(`fs/promises`);
 
 const folder = "node_modules";
 
-const context = "management-api";
-const resourceUrl = `https://eclipse-edc.github.io/Connector/openapi/${context}/${context}.yaml`;
+const CONTEXTS = [
+  {
+    name: "management-api",
+    resourceUrl: "https://eclipse-edc.github.io/Connector/openapi/management-api/management-api.yaml"
+  },
+  {
+    name: "identity-hub-api",
```

Comment:
```suggestion
    name: "identity-api",
```

as this is related to the api, not to the product